service/pom.xml

```diff
@@ -40,6 +40,12 @@
   </scm>
 
   <dependencies>
+    <dependency>
```

Comment:
I don't think the convenience library will work, because this build uses `proc:none`. You'll have to do the registration the manual way (which isn't that much harder).

Reply:
Ha! good catch; fixed.